parent/pom.xml

```diff
@@ -0,0 +1,462 @@
+        <project.build.java.source>${javac.source}</project.build.java.source>
+        <maven.compiler.source>${javac.source}</maven.compiler.source>
+        <ant.build.javac.source>${javac.source}</ant.build.javac.source>
+
+        <javac.target>${javac.source}</javac.target>
+        <project.build.java.target>${javac.target}</project.build.java.target>
+        <maven.compiler.target>${javac.target}</maven.compiler.target>
+        <ant.build.javac.target>${javac.target}</ant.build.javac.target>
+
+        <min.required.maven.version>3.2</min.required.maven.version>
+        <surefire.plugin.version>2.19</surefire.plugin.version>
+            <groovy.major.version>2</groovy.major.version>
+            <groovy.minor.version>4</groovy.minor.version>
+            <groovy.release.version>5</groovy.release.version>
+            <groovy.compliance.level>2.0</groovy.compliance.level>
+        <groovy.version>${groovy.major.version}.${groovy.minor.version}.${groovy.release.version}</groovy.version>
+        <gmaven.plugin.version>1.5</gmaven.plugin.version>
+
+        <ant.version>1.9.6</ant.version>
+        <xmltask.version>1.16</xmltask.version>       
+        <ant-contrib.version>1.0b3</ant-contrib.version>
+        <junit.version>4.12</junit.version>
+    </properties>
+
+    <prerequisites>
+        <maven>${min.required.maven.version}</maven>
+    </prerequisites>
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>junit</groupId>
+                <artifactId>junit</artifactId>
+                <version>${junit.version}</version>
+				<scope>test</scope>
+            </dependency>
+            <dependency>
+                <groupId>org.mockito</groupId>
+                <artifactId>mockito-all</artifactId>
+                <version>1.10.19</version>
+				<scope>test</scope>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
+
+    <licenses>
+        <license>
+            <name>LGPL, version 2.1</name>
+            <url>http://www.gnu.org/licenses/licenses.html</url>
+            <distribution>repo</distribution>
+        </license>
+        <license>
+            <name>ASL, version 2</name>
+            <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+            <distribution>repo</distribution>
+        </license>
+    </licenses>
+
+    <scm>
+        <connection>scm:git:https://github.com/java-native-access/jna</connection>
+        <developerConnection>scm:git:ssh://git@github.com/java-native-access/jna.git</developerConnection>
+        <url>https://github.com/java-native-access/jna</url>
+    </scm>
+
+    <developers>
+        <developer>
+            <id>twall</id>
+            <name>Timothy Wall</name>
+            <roles>
+                <role>Owner</role>
+            </roles>
+        </developer>
+    </developers>
+    
+    <build>
+        <defaultGoal>install</defaultGoal>
```

Comment:
Would not recommend a defaultGoal of "install". I've seen maven projects get into trouble when accidentally requiring an "install" phase, when the "package" phase should be sufficient. I suggest either remove the defaultGoal, or change it to "package". 

PS: I'm working on a PR from you branch with a bunch of the suggested changes